daemon/container.go

```diff
@@ -291,6 +291,22 @@ func (daemon *Daemon) verifyContainerSettings(platform string, hostConfig *conta
 		return nil, nil
 	}
 
+	if hostConfig.Privileged {
+		for _, deviceMapping := range hostConfig.Devices {
+			if deviceMapping.PathOnHost == deviceMapping.PathInContainer {
+				continue
+			}
+			if _, err := os.Stat(deviceMapping.PathInContainer); os.IsNotExist(err) {
+				continue
+			} else {
+				if err == nil {
+					return nil, errors.Errorf("In privileged mode,Path:%s in Container should be different with any path on host ", deviceMapping.PathInContainer)
```

Comment:
Can we clean up some of the grammar here?
Suggestion:
> container device path must be different from any host device path for privileged mode containers

Reply:
Done